packages/api/src/schema.ts

```diff
@@ -1912,6 +1912,93 @@ const schema = gql`
     BAD_REQUEST
   }
 
+  union RulesResult = RulesSuccess | RulesError
+
+  type RulesSuccess {
+    rules: [Rule!]!
+  }
+
+  type Rule {
+    id: ID!
+    name: String
+    query: String!
+    actions: [RuleAction!]!
+    triggerType: RuleTriggerType!
+    createdAt: Date!
+    updatedAt: Date!
+  }
+
+  type RuleAction {
+    type: RuleActionType!
+    value: String
+  }
+
+  enum RuleActionType {
+    ADD_LABEL
+    ARCHIVE
+    MARK_AS_READ
+    SEND_NOTIFICATION
+  }
+
+  enum RuleTriggerType {
+    ON_PAGE_UPDATED
+    CRON
+  }
```

Comment:
@jacksonh Rule trigger type is an enum value of either `ON_PAGE_UPDATED` or `CRON`.
I use `ON_PAGE_UPDATED` not `ON_PAGE_CREATED` because there is no content of the page when it is created before parsing